README.md

```diff
@@ -167,7 +168,9 @@ appbuilder sample clone <Sample>
 
 To list the available sample apps, run `appbuilder sample`.
 
-The Telerik AppBuilder CLI creates a new subdirectory in the current directory and clones the sample inside it.
+The Telerik AppBuilder CLI creates a new subdirectory in the current directory, clones the sample inside it and preserves the existing project configuration of the sample app.
+
+If you want to develop for {{site.wp8}}, make sure to manually update the `WP8PublisherID` and `WP8ProductID` properties to avoid issues when running your app on device. For more information about how to configure your project properties, run `appbuilder prop --help`.
 
```

Comment:
I suggest adding that these ids need to be unique - just to give a reason why they need to be updated. :smiley_cat: